crystal/kmeans.cr

```diff
@@ -52,15 +47,13 @@ def clusters(xs : Array(Point), centroids : Array(Point))
 end
 
 def run(xs : Array(Point))
-  centroids = xs.first(n)
+  centroids = xs.first(N)
 
-  i = 0
-  while i < (iters - 1)
+  ITERS.times do
     centroids = clusters(xs, centroids).map { |l| average(l) }
-    i += 1
   end
 
-  clusters(xs, centroids).map { |l| average(l) }
+  centroids
```

Comment:
While this is equivalent to the previous implementation, this is not equivalent to the scala implementation as it does not run the final `average`